services/requester/requester.go

```diff
@@ -213,7 +183,30 @@ func (e *EVM) SendRawTransaction(ctx context.Context, data []byte) (common.Hash,
 		return common.Hash{}, errs.NewTxGasLimitTooHighError(txMaxGasLimit)
 	}
 
-	if err := models.ValidateTransaction(tx, e.head, e.evmSigner, e.validationOptions); err != nil {
+	head := &types.Header{
+		Number:     big.NewInt(20_182_324),
```

Comment:
why is this block number hardcoded?

Reply:
This one is not really important, as it's no longer used for hard-forks. Ethereum does upgrades with timestamps, instead of block numbers. But this is there for backwards-compatibility. I think this was the Ethereum mainnet latest block number, when we launched Flow EVM on mainnet.

Reply:
@m-Peter please add that as a comment, magic numbers not good :)

Reply:
@j1010001 Good point :+1: 
I added some comments which better describe the intent, in this commit: https://github.com/onflow/flow-evm-gateway/pull/818/commits/45de8d326090f3c0edfae20e9832a540eaa5519e .